projects/js-packages/publicize-components/src/components/generated-image-preview/index.js

```diff
@@ -0,0 +1,68 @@
+import { ThemeProvider } from '@automattic/jetpack-components';
+import { Spinner } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import React, { useEffect, useRef, useState } from 'react';
+import useImageGeneratorConfig from '../../hooks/use-image-generator-config';
+import styles from './styles.module.scss';
+
+/**
+ * Fetches the preview of the generated image based on the post info
+ *
+ * @returns {React.ReactNode} The generated image preview.
+ */
+export default function GeneratedImagePreview() {
+	const [ generatedImageUrl, setGeneratedImageUrl ] = useState( null );
+	const [ isLoading, setIsLoading ] = useState( true );
+
+	const { customText, imageType, imageId } = useImageGeneratorConfig();
```

Comment:
Just looking at this, we're going to need the URL of the image, not the ID. I guess we're able to get that?

Reply:
Yeah, I found it in Social Previews, so I used the same solution